src/main/java/wordcloud/palette/LinearGradientColorPalette.java

```diff
@@ -0,0 +1,102 @@
+package wordcloud.palette;
+
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A LinearGradient ColorPalette
+ * 
+ * @author &#64;wolfposd
+ */
+public class LinearGradientColorPalette extends ColorPalette
+{
+    /**
+     * <p>
+     * Creates a ColorPalette using a linear gradient between the two colors
+     * specified
+     * </p>
+     * <p>
+     * The following example code will generate a gradient between red and blue,
+     * with 18 colors in between red and blue, thus totaling 20 colors:
+     * </p>
+     * <code>
+     * new LinearGradientColorPalette(Color.RED, 18, Color.Blue)
+     * </code>
+     * 
+     * @param color1
+     *            start color
+     * @param betweenC1andC2
+     *            specifies the amount of colors for this gradient between
+     *            color1 and color2
+     * @param color2
+     *            end color
+     */
+    public LinearGradientColorPalette(Color color1, int betweenC1andC2, Color color2)
+    {
+        this.colors = createLinearGradient(color1, color2, betweenC1andC2).toArray(new Color[] {});
+    }
+
+    /**
+     * Creates a ColorPalette using a linear gradient between the three colors
+     * specified
+     * 
+     * @param color1
+     *            start color
+     * @param betweenC1andC2
+     *            number of colors to be generated between color1 and color2
+     * @param color2
+     *            middle color
+     * @param betweenC2andC3
+     *            number of colors to be generated between color2 and color3
+     * @param color3
+     *            end color
+     */
+    public LinearGradientColorPalette(Color color1, int betweenC1andC2, Color color2, int betweenC2andC3, Color color3)
+    {
+        ArrayList<Color> cs = new ArrayList<>();
+
+        for (Color c : createLinearGradient(color1, color2, betweenC1andC2))
+        {
+            cs.add(c);
+        }
+
+        for (Color c : createLinearGradient(color2, color3, betweenC2andC3))
+        {
+            if (c.equals(color2)) // already being added by first gradient
```

Comment:
even for one line if statements I prefer to wrap in { }
Also for trivial if statements (return/break/continues) i don't mine just doing it in one line. 
```java
if (c.equals.color2) { continue; }
```